CONTRIBUTING.rst

```diff
@@ -49,28 +49,41 @@ These are the basic steps needed to start developing on QDarkStyle.
 
    .. code:: bash
 
-          git clone https://github.com/USERNAME/qdarkstyle
-          cd qdarkstyle
+      git clone https://github.com/USERNAME/qdarkstyle
+      cd qdarkstyle
 
 -  Checkout the appropriate branch
 
    .. code:: bash
 
-          git checkout master
+      git checkout master
```

Comment:
@ccordoba12 Could you please change to develop until we have all the resources generated in the server? Until that, I need to generate them in my machine. Tks

Reply:
Done @dpizetta. I also moved the branch of this PR from `master` to `develop`. Is that ok? or do you prefer to keep it in `master`?

Reply:
It is ok, tks!